src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -1513,6 +1512,16 @@ public void fillRankOptionForNewQuestion(int optionIndex, String optionText) {
         fillRankOption(NEW_QUESTION_NUM, optionIndex, optionText);
     }
 
+    public void fillRankMinOptionForQuestion(int qnNumber, String minOption) {
```

Comment:
We now have duplicate methods such as `setMinOptionsToBeRanked` doing similar things?